.github/workflows/publish.yml

```diff
@@ -95,13 +95,34 @@ jobs:
       - name: Verify demo browser npm version and run deployment script
         run: |
           demo_current_version=$(.github/script/get-demo-browser-current-version)
+          demo_current_version=$(echo $demo_current_version | tr -d '^~*')
```

Comment:
Would it be good to do the removal of `!~*` inside the `.github/script/get-demo-browser-current-version` itself?

Reply:
If the script is re-usable and same is required as a output hence checking if it makes sense to get that logic done inside the script itself.

Reply:
I have removed the `tr -d` and have added it to the `get-demo-browser-current-version`. Since, the new script is a node env the code has changed to
```current_version = current_version.replace(/([~^*><=])/g, "");```